src/dsp.jl

```diff
@@ -229,16 +229,16 @@ julia> x = mseq((1,3))              # generate m-sequence with specification (1,
  -1.0
 ```
 """
-function mseq(m, θ=π/2)
+function mseq(m, θ=π / 2)
   knownspecs = Dict(  # known m-sequences are specified as base 1 taps
-       2 => (1,2),          3 => (1,3),          4 => (1,4),          5 => (2,5),
-       6 => (1,6),          7 => (1,7),          8 => (1,2,7,8),      9 => (4,9),
-      10 => (3,10),        11 => (9,11),        12 => (6,8,11,12),   13 => (9,10,12,13),
-      14 => (4,8,13,14),   15 => (14,15),       16 => (4,13,15,16),  17 => (14,17),
-      18 => (11,18),       19 => (14,17,18,19), 20 => (17,20),       21 => (19,21),
-      22 => (21,22),       23 => (18,23),       24 => (17,22,23,24), 25 => (22,25),
-      26 => (20,24,25,26), 27 => (22,25,26,27), 28 => (25,28),       29 => (27,29),
-      30 => (7,28,29,30)
+    2 => (1, 2), 3 => (1, 3), 4 => (1, 4), 5 => (2, 5),
+    6 => (1, 6), 7 => (1, 7), 8 => (1, 2, 7, 8), 9 => (4, 9),
+    10 => (3, 10), 11 => (9, 11), 12 => (6, 8, 11, 12), 13 => (9, 10, 12, 13),
+    14 => (4, 8, 13, 14), 15 => (14, 15), 16 => (4, 13, 15, 16), 17 => (14, 17),
+    18 => (11, 18), 19 => (14, 17, 18, 19), 20 => (17, 20), 21 => (19, 21),
+    22 => (21, 22), 23 => (18, 23), 24 => (17, 22, 23, 24), 25 => (22, 25),
+    26 => (20, 24, 25, 26), 27 => (22, 25, 26, 27), 28 => (25, 28), 29 => (27, 29),
+    30 => (7, 28, 29, 30)
```

Comment:
I would really like to avoid the auto-reformatting of this table with your local preferences. Makes it hard to read, as it loses the alignment that was intended.

Reply:
[38075f9](https://github.com/org-arl/SignalAnalysis.jl/pull/46/commits/38075f9ade73c83bac731f4b9f595115e2cdb5fb)

Reply:
I'll fix the autoformatter so I don't keep mangling code and diffs